packages/core/release/lib/index.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
```

Comment:
I think you need to `git rm -rf packages/core/release`.

Reply:
Yes, this package was removed during the migration to Changesets.